src/registrar/config/settings.py

```diff
@@ -162,7 +162,7 @@
     # django-cors-headers: listen to cors responses
     "corsheaders.middleware.CorsMiddleware",
     # custom middleware to stop caching from CloudFront
-    "registrar.no_cache_middleware.NoCacheMiddleware",
+    "registrar.registrar_middleware.NoCacheMiddleware",
```

Comment:
Was this a 3rd-party update? (I don't know what the diff is between these middlewares)